plugins/file/index.js

```diff
@@ -158,11 +158,10 @@ module.exports = {
       },
       handler: async function (request, h) {
         return new Promise((resolve, reject) => {
-          s3.getObject(
-            {
+          s3Client.send(new GetObjectCommand({
```

Comment:
This could also be simplified by using async/await with a try/catch block to throw the error.